.gitignore

```diff
@@ -0,0 +1,5 @@
+.env
+/wme-sdk-go
```

Comment:
Do we need this bit?

Reply:
It's the go binary name, I'd prefer to keep it in